content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md

```diff
@@ -275,6 +279,30 @@ workers:
         external-dns.beta.kubernetes.io/hostname: corda.example.com
 ```
 
+#### Install the REST Worker Certificate
+
+The REST worker TLS certificate is presented to a client any time a HTTPS connection is made.
+If no specific parameters are provided, a self-signed certificate is used and the connection to the REST Worker is always HTTPS. However, a warning will be emitted into the REST log explaining how to provide parameters for custom TLS certificates.
+The following is required to install a valid TLS certificate:
+* The TLS certificate itself must be signed by a Certification Authority (CA) or an intermediary.
+* A private key corresponding to the public key must be included in the TLS certificate.
```

Comment:
Private key is a highly sensitive information and is not included into TLS certificate which is presented to every client connecting to REST Service. Therefore proposing to change:
```suggestion
* A private key corresponding to the public key included in the TLS certificate.
```